app/assets/javascripts/initializers/initializeHeroBannerClose.js

```diff
@@ -3,6 +3,7 @@
 function initializeHeroBannerClose() {
   let bannerWrapper = document.getElementById('hero-html-wrapper');
   let closeIcon = document.getElementById('js-hero-banner__x');
+  closeIcon.setAttribute('aria-label', 'Close campaign banner');
```

Comment:
I looked through the code, and this seems to be the only reference to `js-hero-banner__x`, so this seems fine adding the aria label here.